doc/nrf/protocols/bt_mesh/ble_mesh_dfu_samples.rst

```diff
@@ -32,12 +32,13 @@ For the complete list of commands, see :ref:`zephyr:bluetooth_mesh_shell_dfd_ser
 Provisioning and configuring the devices
 ========================================
 
-After programming the samples onto the boards, they need to be provisioned into the same Bluetooth mesh network with an external provisioner device.
+After programming the samples onto the boards, they need to be provisioned into the same Bluetooth mesh network with an external provisioner device like described in sample documentation (:ref:`Provisioning the mesh DFU target device <ble_mesh_dfu_target_provisioning>` and :ref:`Provisioning the mesh DFU distributor device <ble_mesh_dfu_distributor_provisioning>`).
```

Comment:
```suggestion
After programming the samples onto the boards, they need to be provisioned into the same Bluetooth mesh network with an external provisioner device.
See the documentation for :ref:`provisioning the mesh DFU target device <ble_mesh_dfu_target_provisioning>` and :ref:`provisioning the mesh DFU distributor device <ble_mesh_dfu_distributor_provisioning>`  for how this is done.
```
It's such a long sentence, that breaking it into two feels better. Not sure this split is the best, though.